trio/_subprocess/tests/test_highlevel_pipes.py

```diff
@@ -0,0 +1,78 @@
+import os
+import pytest
+
+from ... import _core
+from ..unix_pipes import PipeSendStream, PipeReceiveStream, make_pipe
+from ...testing import (wait_all_tasks_blocked, check_one_way_stream)
+
+pytestmark = pytest.mark.skipif(
+    not hasattr(os, "pipe2"), reason="pipes require os.pipe2()"
+)
```

Comment:
Now that we don't use `pipe2` this looks a little odd :-). Maybe `skipif(os.name != "posix", reason=...)` would be a bit clearer. (I think it has the same effect.)